movie.sh

```diff
@@ -1,6 +1,7 @@
 #!bin/bash
 
-readonly TEMP_PATH="/var/folders/3c/gjd471t51px36pjlbr7zggmh0000gn/T/imdb"
+#readonly TEMP_PATH="/var/folders/3c/gjd471t51px36pjlbr7zggmh0000gn/T/imdb"
+readonly TEMP_PATH="$TMPDIR/imdb"
 mkdir $TEMP_PATH      # Making a Folder to Dump Files
```

Comment:
This will work perfectly only if everything works perfectly. Lets say there are no arguments given than it will create a directory and then exit with removing the directory. And now when you run it again, it will throw an error.